monitors/filterWords.js

```diff
@@ -0,0 +1,44 @@
+exports.run = async (client, msg) => {
+  const modlog = msg.guild.settings.modLog;
+  if (msg.deletable) {
+    if (msg.guild.settings.swearWords) {
```

Comment:
Are you aware that swearWords, when inconfigured, is an empty array, and that `[]` is truthy? Use [Array.prototype.length](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length) instead.